### PR TITLE
argon2.0.1 - via opam-publish

### PR DESCRIPTION
**Pull request by Khady**
OCaml bindings to Argon2

Original C implementation: https://github.com/P-H-C/phc-winner-argon2

---
* Homepage: https://github.com/Khady/ocaml-argon2
* Source repo: git+https://github.com/Khady/ocaml-argon2.git
* Bug tracker: https://github.com/Khady/ocaml-argon2/issues

---

Pull-request generated by opam-publish v0.3.1

### CODE REVIEW

**Comment by dsheets** [comment]
It looks like this package requires `ppx_deriving` of at least 1.0.

File: packages/argon2/argon2.0.1/descr [hunk 1]
```diff
@@ -0,0 +1,3 @@
+OCaml bindings to Argon2
+
+Original C implemenation: https://github.com/P-H-C/phc-winner-argon2
```
Comment (dsheets) [inline packages/argon2/argon2.0.1/descr]: s/implemenation/implementation/

**Comment by Khady** [comment]
Yes, I just saw the errors. Sorry about the noise. Working on the fix.

**Comment by Khady** [comment]
Seems good now. Thanks for the typo.

**Pull request merged**